plugin/NERD_commenter.vim

```diff
@@ -986,19 +987,20 @@ function s:CommentLinesSexy(topline, bottomline)
         call cursor(a:topline, 1)
         execute 'normal! O'
         let theLine = repeat(' ', leftAlignIndx) . left
-
```

Comment:
Please don't remove these blank lines between blocks (here plus L995 and L1001). They make it easier to see what code the comments apply to and to jump around by paragraph using vim's text object bindings. Thanks.

Reply:
I've done the changes you requested. Add the doc. Remove the debug lines. Restore the blank lines.